src/components/Clickable/Clickable.tsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React, { AriaRole, CSSProperties, forwardRef, MouseEventHandler, ElementType, ReactNode, useRef } from "react";
+import { backwardCompatibilityForProperties } from "../../helpers/backwardCompatibilityForProperties";
+import cx from "classnames";
+import useMergeRefs from "../../hooks/useMergeRefs";
+import { BEMClass } from "../../helpers/bem-helper";
+import { useKeyboardButtonPressedFunc } from "../../hooks/useKeyboardButtonPressedFunc";
+import { ELEMENT_TYPES, getTestId } from "../../utils/test-utils";
+import VibeBaseButtonComponentProps from "../../types/VibeBaseButtonComponentProps";
+import VibeComponent from "../../types/VibeComponent";
+import "./Clickable.scss";
+
+const CSS_BASE_CLASS = "monday-style-clickable";
+const bemHelper = BEMClass(CSS_BASE_CLASS);
+
+export interface ClickableProps extends VibeBaseButtonComponentProps<HTMLElement> {
+  elementType?: ElementType;
+  children?: string | ReactNode | null | [ReactNode];
```

Comment:
Did you mean array of `ReactNode`?

```suggestion
  children?: string | ReactNode | null | ReactNode[];
```

What you wrote originally means "an array with a single item that is a `ReactNode`", meaning if an array of 2 or more `ReactNode`s is passed, TS would throw an error.